internal/exec/engine.go

```diff
@@ -84,17 +84,23 @@ func (e Engine) Run(stageName string) bool {
 		e.logReport(r)
 		if err != nil && err != providers.ErrNoProvider {
 			e.Logger.Crit("failed to acquire default config: %v", err)
-			return false
+			return err
 		}
 	default:
 		e.Logger.Crit("failed to acquire config: %v", err)
-		return false
+		return err
 	}
 
 	e.Logger.PushPrefix(stageName)
 	defer e.Logger.PopPrefix()
 
-	return stages.Get(stageName).Create(e.Logger, e.Root, *e.Fetcher).Run(config.Append(baseConfig, config.Append(systemBaseConfig, cfg)))
+	if err = stages.Get(stageName).Create(e.Logger, e.Root, *e.Fetcher).Run(config.Append(baseConfig, config.Append(systemBaseConfig, cfg))); err != nil {
+		// e.Logger could be nil
+		fmt.Fprintf(os.Stderr, "%s failed", stageName)
+		return err
+	}
+	e.Logger.Info("passed")
```

Comment:
Add stage name here like in the failed message.

Reply:
okay fixed the new line in packet.go on [line 131](https://github.com/coreos/ignition/pull/566/commits/d604cd9e2a6f6c0effb90f053a79573baaae47f0#diff-1e02369f5d00ece3822f5641a824b460R113) and changed `"passed"` to `"%s passed", stageName` [see here](https://github.com/coreos/ignition/pull/566/commits/ce93f0a86b1032ce52d85b2a8e8e7d13d2a429d5#diff-e3491b7c0e160811ce6761d5a1117558R102).